backend/src/incidents/views.py

```diff
@@ -197,7 +201,17 @@ def post(self, request, format=None):
             print("errors: ", serializer.errors)
 
         if serializer.is_valid():
+            ''' Function to generate refId for requests '''
+            profile = Profile.objects.get(user=request.user)
+
+            today = datetime.now().replace(hour=0, minute=0, second=0)
```

Comment:
we have to make the count from the begining of the year, since we have only the 'year' on the id.